packages/wdio-cli/src/commands/config.ts

```diff
@@ -65,6 +65,16 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
         if (!hasPackage('ts-node')) {
             packagesToInstall.push('ts-node', 'typescript')
         }
+        if (!hasFile('tsconfig.json')){
+            const config = {
+                'compilerOptions': {
+                    'types': ['node', 'webdriverio/async', frameworkPackage.package],
+                    'target': 'ES6'
+                }
+            }
+
+            await fs.promises.writeFile(path.join(process.cwd(), 'tsconfig.json'), JSON.stringify(config, null, 2))
```

Comment:
An indentation of 4 is used for the Babel config, any reason to change that for Typescript?

Reply:
I will change to 4 , thank you so much @erwinheitzman  for the review 🎉